### PR TITLE
Feature/multi search

### PR DESCRIPTION
**Pull request by kjaym**
@maxhutch Adds support for multi-search, which is coming soon.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
 setup(name='citrination-client',
-      version='1.2.12',
+      version='1.4.0',
```
Comment (maxhutch) [inline setup.py]: We're bumping versions in release branches now.

**Comment by maxhutch** [comment]
This isn't so much related to this PR in particular, but can we start adding tests with new functionality?

**Comment by kjaym** [comment]
Like we talked about, the functionality is not ready to tests. Adding an issue.

**Comment by maxhutch** [comment]
LGTM

**Pull request merged**